configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Configuring Root Certificates"
+lastmod = "2022-06-10T14:22:00-06:00"
++++
+# Configuring Root Certificates
+
+Mange local SSL/TLS Certificate Authority (CA) certificates with any of the desktop environments supported by Solus.
+
+## Public vs. Local Certificates
+
+Public SSL certiicates are included in Solus system packages and are regularly updated with normal package updates.
+
+There are many reasons you may want to install a custom, self-signed, or certificate provided by your local systems administrator.
+
+The topic of SSL and TLS certificates for protecting network resources is a deep and complex topic. Many great resources are publicly available for fundamental usage and workflows. This article will describe the default utilities and NSS database location in Solus.
+
+## Basic Requirements
+
+This article assumes basic familirarity with using the command line. 
+
+A web browser and desktop environment like Budgie can be very useful in testing a certificate served for a network resource.
+
+The creation and validation of a certiicate is outside the scope of this article and you must already have a signed certiicate. 
+
+Although not strictly required, normal certificate usage is in the context of some network, and often the public internet. 
```

Comment:
I'm not really sure what this sentence is trying to say.

Reply:
I think the intent here is to say something link "Normally certificates are used by secured network connections on the internet and corporate networks. But certificate can also be used for e-mail signing and other use cases."

Reply:
Thanks for the feedback and idea! My hope was to call out that there are different use-cases, and have additional "Basic Requirements" not covered. I tried to clarified this with "These instructions assume usage for secure network connections on the internet or corporate networks, but may also be used for other purposes like signing email."